doc/source/index.rst

```diff
@@ -124,22 +124,23 @@ Ray provides Python, Java, and *EXPERIMENTAL* C++ API. And Ray uses Tasks (funct
     .. code-block:: shell
 
       bazel build //cpp:ray_cpp_pkg
-      bazel build //cpp/example:example
+      cd cpp/example/
```

Comment:
Can we avoid this, just use one command.

Reply:
We need this because example has been a independent bazel project. Do you mean like this `cd cpp/example/ && bazel build //:example`?

Reply:
How about bazel build //cpp/example:example?
Don't need to goto a sub dir.

Reply:
If we use `build //cpp/example:example`, we can not ensure this project is independent. Just like a sub-project under ray project.